rokka/sourceimages.go

```diff
@@ -122,3 +126,36 @@ func (c *Client) CreateSourceImageWithMetadata(org, name string, data io.Reader,
 
 	return result, err
 }
+
+func (c *Client) DynamicMetadataResponseHandler(v interface{}, resp *http.Response) error {
+	if resp.StatusCode == 204 {
+		dynamicMetadataResponse, ok := v.(*PutDynamicMetadataResponse)
+
+		if ok {
```

Comment:
that basicly can't happen that it's not `ok`. It's still good to check it, but if it's not ok you should `panic()`